meta/etcd.go

```diff
@@ -0,0 +1,100 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package meta
+
+import (
+	"context"
+	"time"
+
+	"cqfn.org/degitx/discovery"
+	"cqfn.org/degitx/misc"
+
+	ma "github.com/multiformats/go-multiaddr"
+	"go.etcd.io/etcd/clientv3"
+)
+
+type etcd struct {
+	Endpoints []string
```

Comment:
@stepanvalyavskiy this field could be private (`endpoints`), since the struct is private too

Reply:
sure, done